data/scripts.js

```diff
@@ -1028,6 +1204,22 @@ let BattleScripts = {
 	targetTypeChoices(targetType) {
 		return CHOOSABLE_TARGETS.has(targetType);
 	},
+
+	// @ts-ignore I can add two numbers and return a number, ok?
```

Comment:
```suggestion
```

This is a really dangerous place for a `ts-ignore`. I'd just cast the result as `any`.